gnd/src/main/java/com/google/android/gnd/persistence/geojson/GeoJsonExtent.java

```diff
@@ -17,49 +17,44 @@
 package com.google.android.gnd.persistence.geojson;
 
 import com.google.android.gms.maps.model.LatLng;
-import com.google.android.gms.maps.model.LatLngBounds;
-import java8.util.Optional;
+import com.google.common.collect.ImmutableList;
 import javax.annotation.Nullable;
 import org.json.JSONArray;
 
-public class GeoJsonExtent {
+class GeoJsonExtent {
 
   private final GeoJsonGeometry geometry;
-  private final Optional<LatLng[]> coordinates;
 
   @Nullable
   GeoJsonExtent(GeoJsonGeometry geometry) {
     this.geometry = geometry;
-    this.coordinates = getPolygonCoordinates();
   }
 
-  Optional<LatLngBounds> getBounds() {
-    return this.coordinates.map(cs -> new LatLngBounds(cs[0], cs[2]));
+  ImmutableList<LatLng> getVertices() {
+    return getPolygonVertices();
```

Comment:
Can we get rid of this method and expose or rename `getPolygonVertices()`

Reply:
Sgtm, done!